v2.1/import-data.md

```diff
@@ -77,10 +77,94 @@ For reference, CockroachDB uses these defaults:
 - `[port]`: **26257**
 - `[user]`: **root**
 
+
+## Import from MySQL Dump
+
+{% include experimental-warning.md %}
+
+<span class="version-tag">New in v2.1:</span> This section has instructions for getting data from MySQL dump files into CockroachDB using [`IMPORT`](import.html). It uses the [employees data set](https://github.com/datacharmer/test_db) that is also used in the [MySQL docs](https://dev.mysql.com/doc/employee/en/).
+
+- [Step 1. Dump the MySQL database](#step-1-dump-the-mysql-database)
+- [Step 2. `IMPORT` the dump files](#step-2-import-the-dump-files)
+
+### Step 1. Dump the MySQL database
+
+Issue a command like the following (modified as needed for your environment):
+
+{% include copy-clipboard.html %}
+~~~ sh
+$ mysqldump -uroot --tab=/tmp employees --fields-enclosed-by='"' --fields-escaped-by='\'
+~~~
+
+Always specify an escape character when running `mysqldump --tab`. If your data happens to include any of the special characters, the dumped data cannot be decoded correctly if they are not escaped. We also recommend specifying an enclose character.
```

Comment:
In https://docs.google.com/document/d/16yWcR4CiOGhYaODtsyA1X28oFTgtCvr5AUWvMBUOSic/edit we say it's "probably a good idea", so I went with it since this is "alpha" :-)  I know @dt wrote  in https://github.com/cockroachdb/cockroach/pull/25287 that 

<blockquote>
MySQL's OUTFILE (as used in mysqldump --tab) can be configured to almost  
CSV, but its escaping and field enclosing behavior does not quite match  
the CSV spec.
</blockquote>

I suspect it's related to that?  Not sure though, perhaps David has more detail.